rtic-time/Cargo.toml

```diff
@@ -20,6 +20,7 @@ license = "MIT OR Apache-2.0"
 critical-section = "1"
 futures-util = { version = "0.3.25", default-features = false }
 rtic-common = { version = "1.0.0", path = "../rtic-common" }
+atomic-polyfill = "1"
```

Comment:
`atomic-polyfill` is deprecated. Use `portable-atomic` instead.

Reply:
While this is true, `rtic` and `rtic-monotonics` already uses `atomic-polyfill`. Should we switch those as well, or use `atomic-polyfill` for now?

Reply:
I switched `rtic-monotonics` and `rtic-time` to `portable-atomic`. I was scared of touching `rtic` itself, though, as I'm not sure if those are exposed and hence whether it would be a breaking change.

Reply:
Actually I couldn't make it work with `portable-atomic`, so I switched back to `atomic-polyfill` for now.

Reply:
Background: https://github.com/taiki-e/portable-atomic/issues/133
Either we use `atomic-polyfill` here, or we wait for this issue to be resolved.

Reply:
Fix for now: remove `atomic-polyfill` completely and replace it with `core::sync::atomic`s.